src/alexapi/triggers/platformtrigger.py

```diff
@@ -54,11 +54,11 @@ def platform_callback(self, platform_continuous_callback=None):
 				long_press_thread.start()
 
 	def continuous_callback(self):
-		if self._platform_continuous_callback:
-			return self._platform_continuous_callback()
-		else:
+		if not self._platform_continuous_callback:
 			return False
 
+		return self._platform_continuous_callback()
```

Comment:
Because the relation is strictly this: `platform <---> trigger <---> recording` and the platform shouldn't get in touch with recording. Also, we use the callback in `long_press()` in the trigger.